GoogleMapsApi/Entities/TimeZone/Response/TimeZoneResponse.cs

```diff
@@ -30,7 +30,7 @@ internal string StatusStr
         /// DstOffset: the offset for daylight-savings time in seconds. This will be zero if the time zone is not in Daylight Savings Time during the specified timestamp.
         /// </summary>
         [DataMember(Name = "dstOffset")]
-        public double OffSet { get; set; }
+        public double DstOffSet { get; set; }
```

Comment:
Right.
I really forgot this strategy. I will fix it soon.
One question about it: What is the ideal time to let the 'OffSet' variable?